deploy/olm-catalog/service-telemetry-operator/manifests/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -127,6 +127,11 @@ spec:
                                 - persistent
                                 type: string
                             type: object
+                          version:
+                            description: Version of ElasticSearch to deploy. Versions
```

Comment:
Same suggestion here, but the UI makes it a pain to "suggest" it on a multi-line

Reply:
This one is generated from the source file you previously commented on.

```
operator-sdk generate bundle --channels unstable --default-channel unstable
```